Assets/Shaders/BlurShader/GaussianBlur.cginc

```diff
@@ -0,0 +1,23 @@
+void GaussianBlur_float(UnityTexture2D Texture, float2 UV, float Blur, UnitySamplerState Sampler, out float3 Out_RGB, out float Out_Alpha)
```

Comment:
```suggestion
public void GaussianBlur_float(UnityTexture2D Texture,
                               float2 UV,
                               float Blur,
                               UnitySamplerState Sampler,
                               out float3 Out_RGB,
                               out float Out_Alpha)
```

1. Weet niet zeker welke acced modiefer nodig is. Maar moet er wel bij staan
2. Check ene laatste [convention](https://github.com/Team-Swamp/Codename-BDLMTW/wiki/Code-convention#functions) van dit kopje.